src/Moryx.Maintenance.Web/Moryx.Maintenance.Web.csproj

```diff
@@ -1,5 +1,12 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor" InitialTargets="BuildUI">
```

Comment:
Is this still needed?

Reply:
And this is required as well, has propably something todo with the point in time when msbuild checks if the directory exists, but never confirmed that 🤔